.github/ISSUE_TEMPLATE/1_plugin_issue.yml

```diff
@@ -0,0 +1,74 @@
+name: Plugin issue
+description: One of Streamlink's plugins doesn't work correctly
+title: "plugins.plugin-name-here: A very brief summary of what's broken"
+labels:
+  - plugin issue
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thanks for reporting a plugin issue!
+
+        Plugin issues describe broken functionality within a plugin's code base, eg. the streaming site has made breaking changes, streams don't get resolved correctly, or authentication has stopped working, etc.
+
+        PLEASE DON'T IGNORE this template and fill in all the required details.
+        Issues that don't adhere to our request will be closed and ignored.
```

Comment:
I think you should bold this whole line **Issues that don't adhere to our request will be closed and ignored.** for all of the templates.